front/src/RefboxButton.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+/**
+ * A button part of the refbox. Does a callback to its parent to handle onClick events
+ * Author: S.A. Tanja
+ * Author: G.W. van der Heijden
+ *
+ * Props:
+ *  - teamColor: the team color of the of button (cyan or magenta).
+ *  - tag: the tag of the button according to the TRC.
+ */
+const RefboxButton = props => {
+  return (
+    <Button
+      teamColor={props.teamColor}
+      onClick={() => {
+        props.onClick(props);
+      }}
+    >
+      {props.tag}
+    </Button>
+  );
+};
+
+/**
+ * A styled component defining the style of the refbox buttons
+ */
+const Button = styled.button`
+  background-color: ${props => props.teamColor};
+  width: 20.6rem;
+  height: 20.6rem;
+  border: 0.25rem solid;
```

Comment:
Do we make the border width dependent on the window size?

Reply:
No